doc/release-notes/9356-rate-limiting.md

```diff
@@ -0,0 +1,20 @@
+## Rate Limiting using JCache (with Hazelcast as a provider)
```

Comment:
Technically, we're not using JCache here, just an IMDG provided by Payara using Hazelcast.

Reply:
Switched to use javax.cache.Cache class